app/Modules/Group/groups.php

```diff
@@ -109,14 +109,8 @@
             'name' => 'annotator',
             'display_name' => 'Annotator',
             'description' => 'Annotator',
-        ],
+        ],        
         
-        'private-member' => [
```

Comment:
In the future, changes like this that partially revert portions of another commit on the branch would be better handled prior to review by doing an interactive rebase and using "squash" or "fixup" to combine with the prior commit it is modifying. That way a code reviewer doesn't have to see the code present in one commit and then notice its deletion later. Let me know if this is not clear and I can walk you through the process.